javalib/src/main/scala/java/nio/file/Path.scala

```diff
@@ -35,3 +35,20 @@ trait Path extends Comparable[Path] with Iterable[Path] with Watchable {
   def toString(): String
   def toUri(): URI
 }
+
```

Comment:
1) I think it worthwhile to create another Issue which
    says that the 'trait' part of Path.scala should be   
    investigated to see if it can be brought up to Java 11,
    especially implementing the default methods.
    Implementing default methods might mean removing
    such code in some other files.

     If part of the file is going to be Java 11, it would be nice
     if the entire file, within reason & available resources, 
     were Java 11 (and noted as such).

2) I think the block comment is better placed in   
    "Paths.scala" (with an 's') just above the methods it
    is describing.

     I would suggest that 'Path.of()' becomes available in
     Java 11 but the Paths get messages become marked
    as "to be removed" much later (Java 17?).  Do they still
    exist in Java 19 or 20?  Are they marked as Deprecated?

     I recommend a single line here, such as:
     "// Introduced in Java 11"